server/core/runtime/pre_workflow_hook_runner.go

```diff
@@ -47,8 +51,20 @@ func (wh DefaultPreWorkflowHookRunner) Run(ctx models.WorkflowHookCommandContext
 	if err != nil {
 		err = fmt.Errorf("%s: running %q in %q: \n%s", err, command, path, out)
 		ctx.Log.Debug("error: %s", err)
-		return "", err
+		return "", "", err
+	}
+
+	var cmdOut []byte
+	var errOut error
+	if _, err := os.Stat(outputFilePath); err == nil {
+		cmdOut, errOut = os.ReadFile(outputFilePath)
+		if errOut != nil {
+			err = fmt.Errorf("%s: running %q in %q: \n%s", err, command, path, out)
+			ctx.Log.Debug("error: %s", err)
+			return "", "", err
+		}
 	}
+
 	ctx.Log.Info("successfully ran %q in %q", command, path)
-	return string(out), nil
+	return string(out), strings.Trim(string(cmdOut), "\n"), nil
```

Comment:
What's the difference between out and cmdOut?

Reply:
Thank you for the response.

Would you mind adding that in as a comment for the new `out` input var? At the moment, the naming convention of both are confusing to me. It's not clear from reading the code without having to dive in further.

Reply:
Sure, will do!